src/services/identity/CollaboratorsService.ts

```diff
@@ -157,15 +157,8 @@ class CollaboratorsService {
       return new NotFoundError(`Site does not exist`)
     }
 
-    // 3. Check if valid user exists
-    const user = await this.usersService.findByEmail(email)
-    if (!user) {
-      // Error - user with a valid gov email does not exist
-      logger.error(`create collaborators error: user ${email} is not valid`)
-      return new NotFoundError(
-        `This user does not have an Isomer account. Ask them to log in to Isomer and try adding them again.`
-      )
-    }
+    // 3. Retrieve or create user if user doesn't already exist
+    const user = await this.usersService.findOrCreateByEmail(email)
```

Comment:
won't happen - email now is from sgid

Reply:
Yeah this is true - had a chat earlier and we decided that a quick fix for this would be to periodically go through the user table and remove users with no associated mobile numbers (since this means that have never made any cms edits before as you are blocked on doing anything until you fill in contact details)